westack/tests/westack_datasource_test.go

```diff
@@ -54,6 +54,13 @@ func Test_DatasourceClose(t *testing.T) {
 	err = ds.Close()
 	assert.NoError(t, err)
 
+	// Based on suggestion from @tanryberdi: https://github.com/fredyk/westack-go/pull/480#discussion_r1312634782
+	// Attempt to perform a query. We don't mind the queried collection because the client
+	// is disconnected anyway
+	result, err := ds.FindMany("unknownCollection", nil)
+	assert.Errorf(t, err, "client is disconnected")
+	assert.Nil(t, result)
```

Comment:
@tanryberdi Do you think ds.Close() is well tested now with these additional checks?

Reply:
yes @fredyk, it was great now, sorry for disturbing :D, btw, I will try to add test case for `FindDataSource` functions, you can assign to me

Reply:
Great! Just assigned that other one #493 to you. Thanks very much for your collaboration :smiley: